api/integrations/launch_darkly/models.py

```diff
@@ -15,7 +15,7 @@ class LaunchDarklyImportStatus(TypedDict):
     requested_environment_count: int
     requested_flag_count: int
     result: NotRequired[Literal["success", "failure"]]
-    error_message: NotRequired[str]
+    error_messages: list[str]
```

Comment:
I found it easier to default it to an empty array rather than not required. As there is no logic that benefits from `None` over `[]`.

That's how we simplify the optionality of repeated fields in our scala codebase.

Reply:
Nice, agreed 👍